sdk/translation/test-resources.bicep

```diff
@@ -52,21 +60,34 @@ resource storageAccount 'Microsoft.Storage/storageAccounts@2023-05-01' = {
   location: location
   sku: {
     name: 'Standard_RAGRS'
+    tier: 'Standard'
   }
   kind: 'StorageV2'
   properties: {
     networkAcls: networkAcls
     supportsHttpsTrafficOnly: true
+    allowSharedKeyAccess: false
     encryption: encryption
-    accessTier: 'Cool'
+    accessTier: 'Hot'
     minimumTlsVersion: 'TLS1_2'
   }
 }
 
+// Role assignment to grant Storage Blob Data Contributor role to the Cognitive Services(Translator) Account Managed Identity
+resource storageBlobDataContributorRoleAssignment 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
+  name: guid(storageAccount.id, cognitiveServicesAccount.id, storageBlobDataContributorRoleId)
+  properties: {
+    roleDefinitionId: resourceId('Microsoft.Authorization/roleDefinitions', storageBlobDataContributorRoleId)
+    principalId: cognitiveServicesAccount.identity.principalId
+    principalType: 'ServicePrincipal'
+  }
+  scope: storageAccount
+}
+
 // Outputs
 output TEXT_TRANSLATION_API_KEY string = listKeys(cognitiveServicesAccount.id, apiVersion).key1
 output TEXT_TRANSLATION_ENDPOINT string = cognitiveServicesAccount.properties.endpoint
 output TEXT_TRANSLATION_CUSTOM_ENDPOINT string = 'https://${baseName}${customEndpointSuffix}'
 output TEXT_TRANSLATION_REGION string = location
 output DOCUMENT_TRANSLATION_ENDPOINT string = 'https://${baseName}${dtEndpointSuffix}'
-output DOCUMENT_TRANSLATION_STORAGE_NAME string = storageAccount.name
+output DOCUMENT_TRANSLATION_STORAGE_NAME string = storageAccountName
```

Comment:
nitpick: is this necessary?

Reply:
not really, but since we already have a var , wanted to use it.